core/controllers/profile_test.py

```diff
@@ -236,11 +237,23 @@ def test_get_profile_picture_valid_username(self):
         response_dict = self.get_json(
             '%s%s' % (self.PROFILE_PIC_URL, self.USERNAME)
         )
-        # Although the user has a valid username, they have not yet supplied
-        # a profile picture.
-        self.assertIsNone(
+        self.assertIsNotNone(
```

Comment:
Maybe add a comment: 

If the user has not supplied a profile picture, one will still be generated by default.